src/cli-constant.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+const booleanOptionNames = [
+  "use-tabs",
+  "semi",
+  "single-quote",
+  "bracket-spacing",
+  "jsx-bracket-same-line",
+  // Deprecated in 0.0.10
+  "flow-parser"
+];
+
+const stringOptionNames = [
+  "print-width",
+  "tab-width",
+  "parser",
+  "trailing-comma"
+];
+
+const options = {
+  boolean: [
+    "write",
+    "stdin",
+    // The supports-color package (a sub sub dependency) looks directly at
+    // `process.argv` for `--no-color` and such-like options. The reason it is
+    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
+    // See https://github.com/chalk/supports-color/#info for more information.
+    "color",
+    "list-different",
+    "help",
+    "version",
+    "debug-print-doc",
+    "debug-check",
+    "with-node-modules"
+  ].concat(booleanOptionNames),
+  string: [
+    "cursor-offset",
+    "range-start",
+    "range-end",
+    "stdin-filepath",
+    "config",
+    "find-config-path",
+    "ignore-path"
+  ].concat(stringOptionNames),
+  default: {
+    color: true,
+    "ignore-path": ".prettierignore"
+  },
+  alias: {
+    help: "h",
+    version: "v",
+    "list-different": "l"
+  },
+  unknown: param => {
+    if (param.startsWith("-")) {
+      console.warn("Ignored unknown option: " + param + "\n");
+      return false;
+    }
+  }
+};
+
+const usage =
```

Comment:
I think it would be nice to convert this into a template literal. (Actually, if you feel like it, you could go through the CLI files and replace `"Foo " + bar + " baz"` with template literals.)